homework/index.js

```diff
@@ -29,12 +31,115 @@
     return elem;
   }
 
-  function renderRepoDetails(repo, ul) {
-    createAndAppend('li', ul, { text: repo.name });
+  function addCont(login, img, contributeNumber, parent, link) {
+    const tr = createAndAppend('tr', parent);
+    const imageColumn = createAndAppend('th', tr);
+    createAndAppend('img', imageColumn, {
+      src: img,
+      class: 'contributorImg',
```

Comment:
By convention, CSS identifiers (class names, ids) should be in **kebab-case**, i.e. all lowercase and words separated by hyphens: `contributor-img`.  Please check the rest of your code for similar issues.